frontend/public/components/stateful-set.jsx

```diff
@@ -26,7 +26,7 @@ const Row = props => <WorkloadListRow {...props} kind={kind} actions={menuAction
 const Details = ({obj: ss}) => <React.Fragment>
   <div className="co-m-pane__body">
     <SectionHeading text="StatefulSet Overview" />
-    <ResourceSummary resource={ss} showNodeSelector={false} />
+    <ResourceSummary resource={ss} showPodSelector={true} />
```

Comment:
I know this isn't what was there before, but stateful sets can have a node selector.

```suggestion
    <ResourceSummary resource={ss} showPodSelector showNodeSelector />
```